tools/add_test_users.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# This script add useraccounts to the system for testing Access Control Lists in iceoryx
+# Use at your own risk!
+
+set -e
+
+#====================================================================================================
+#==== Step : Create local test users and groups for testing access control  =========================
+#====================================================================================================
+
+
+if [ $(getent group roudi_test1) ] && [ $(getent group roudi_test2) ] && [ $(getent group roudi_test3) ]; then
```

Comment:
Just adding all users if just one is missing, might also to problems. It must be checked if the specific user is already existing before adding it. Below, there is a `USERS` array which could also be used to check if the user is already existing and if not create one. The sudo check could be done right before the user is added.